packages/core/src/browser/fetchObservable.ts

```diff
@@ -113,5 +114,15 @@ function afterSend(
       observable.notify(context)
     }
   }
-  responsePromise.then(monitor(reportFetch), monitor(reportFetch))
+
+  const reportFetchOnPayloadComplete = (response: Response) => {
+    const cloneResponse = response.clone()
+    if (cloneResponse.body) {
+      readLimitedAmountOfBytes(cloneResponse.body, Number.POSITIVE_INFINITY, () => reportFetch(response), false)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ ‏clone -> cloned and use body directly
```suggestion
    const clonedBody = response.clone().body
    if (clonedBody) {
      readLimitedAmountOfBytes(clonedBody, Number.POSITIVE_INFINITY, () => reportFetch(response), false)
```